### ISSUE TITLE
Move add action on Trigger inside Trigger Detail Fragment

### ISSUE THREAD

**Issue by KeenWarrior** [hawkular/hawkular-android-client#113]
After #112 is complete. Move *Add* action which correspond to adding trigger in favourite triggers, from Triggers Fragment  inside Trigger Detail Fragment

**Comment by sauravvishal8797** [comment]
@garg-anuj Would like to work on this.

**Comment by shubhamvashisht** [comment]
@garg-anuj  #112 is complete. i would like to do this.

**Comment by pg301** [comment]
@garg-anuj I would like to work on it as well.

**Comment by KeenWarrior** [comment]
@pg301 You go for it. 
@shubhamvashisht I am creating another issue for putting a toggle to turn and off a trigger.

**Comment by shubhamvashisht** [comment]
alright  @garg-anuj

**Comment by pg301** [comment]
@garg-anuj Sure! Thanks.

**Comment by pg301** [comment]
![demo](https://cloud.githubusercontent.com/assets/10026907/23762735/04dfa606-051d-11e7-8370-ea95e34c88ed.gif)

**Comment by pg301** [comment]
![demo 1](https://cloud.githubusercontent.com/assets/10026907/23762906/a44507a4-051d-11e7-9a2d-4a2b294b25f7.gif)
@anuj1708 Is this what was needed? Please let me know if you want some changes.

**Comment by KeenWarrior** [comment]
@pg301 Exactly. Pretty cool. Great work. Just make sure 
- reduce size by .2
- Its wrapped in linear layout 
- move it to top

So that other things can be added to fragment easily

**Comment by pg301** [comment]
@anuj1708 Should I change the size of the icon or text? Yeah this is wrapped in Linear Layout and I will move it to the top. Also I wanted to mention that I have use used vectorDrawables as mentioned by @danielpassos. I will do the required changes and open a PR.

**Comment by KeenWarrior** [comment]
@pg301 Amm. Let size be the same for now. We will alter that later is needed. Rest is good. PR is most welcome

**Comment by pg301** [comment]
@garg-anuj I will be commenting out the existing code for adding a trigger inside TriggerFragment. @shubhamvashisht might want to have a look at it when making the toggle.

**Comment by KeenWarrior** [comment]
@pg301 okay. go on

**Comment by danielpassos** [comment]
Talking about UX I don't wanna go to a new screen only to _favorite_  something. Just wondering why not add the _heart_ icon left of the menu (inline)?

**Comment by pg301** [comment]
@danielpassos @garg-anuj Should I keep the functionality of _Adding_ inside TriggerFragment only if there is no need of opening a new screen. I can try replacing the pop up menu button with the animated heart or star.

**Comment by KeenWarrior** [comment]
@danielpassos There gonna come a lot more functionality of editing different fields of trigger in detailed trigger fragment. This issue is just starting of that

**Comment by pg301** [comment]
@garg-anuj can I send a PR then?

**Comment by KeenWarrior** [comment]
@pg301 If work is already complete then go on and place PR but I will need to talk to @danielpassos before merging :)

**Comment by danielpassos** [comment]
> @danielpassos There gonna come a lot more functionality of editing different fields of trigger in detailed trigger fragment. This issue is just starting of that

@garg-anuj Ok. I got your point. So... if _detail_ is not a screen only for favorite we can go for it.

@pg301 Let's keep it in _detail_ screen for now
